libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/kotlin/file/EnumResponseAdapterBuilder.kt

```diff
@@ -65,6 +65,7 @@ internal class EnumResponseAdapterBuilder(
 
     return TypeSpec
         .objectBuilder(layout.enumResponseAdapterName(name))
+        .maybeAddJsExport(context)
```

Comment:
Is this needed since enums can't be exported to JS? Or do I miss something there?